scaleway/provider.go

```diff
@@ -212,6 +212,7 @@ func Provider() terraform.ResourceProvider {
 			"scaleway_security_group_rule":      resourceScalewaySecurityGroupRule(),
 			"scaleway_volume":                   resourceScalewayVolume(),
 			"scaleway_volume_attachment":        resourceScalewayVolumeAttachment(),
+			"scaleway_container_registry":       resourceScalewayContainerRegistry(),
```

Comment:
We will name it `scaleway_registry_namespace_beta`.

We want to respect this pattern for all new resources: `scaleway_{productName}_{resourceName}[_beta]`. For every new resource, we prefer to release it with `_beta` suffix and remove it when we will be confident about the maturity of the resource.

Note that we have to rename:
- All functions: CRUD, helpers, tests, etc
- Documentation
- Files

Sorry for the inconvenience. 😕

Reply:
I hope I have covered all of the cases. If there are more please comment on them and I will change it.